apps/condo/domains/common/components/Comments/index.tsx

```diff
@@ -32,127 +51,382 @@ const Container = styled.aside<IContainerProps>`
         if (isSmall) {
             return 'margin: 0 -20px -60px;'
         } else {
-            return 'height: calc(100vh - 120px);'
+            return 'height: calc(100vh - 100px);'
         }
     }}
 
-  > * {
-    padding: 24px
-  }
-
   display: flex;
   flex-flow: column nowrap;
   align-content: space-between;
 `
-const Head = styled.div`
-  border-bottom: solid thin ${colors.lightGrey};
+const Head = styled.div<{ isTabsHidden: boolean }>`
+  padding: ${({ isTabsHidden }) => isTabsHidden ? '24px' : '24px 24px 0 24px'};
+  border-bottom: ${({ isTabsHidden }) => isTabsHidden ? 'solid thin ' + colors.inputBorderGrey : 'none'}; 
   font-style: normal;
   font-weight: bold;
   font-size: 20px;
   line-height: 28px;
-  margin-bottom: -24px;
 `
 const Body = styled.div`
+  padding: 12px 24px 0;
   overflow-y: scroll;
   flex: 1 1 auto;
-  padding-top: 12px;
 `
-const Footer = styled.div<{
-    hasComments?: boolean
-}>`
-  ${({ hasComments }) => hasComments ? 'border-top: solid thin #D9D9D9;' : ''}
+const Footer = styled.div`
+  border-top: 1px solid ${colors.inputBorderGrey};
+  padding: 8px;
 `
 const EmptyContainer = styled.div`
   text-align: center;
   flex: 1 1;
   display: flex;
   align-items: center;
+  justify-content: center;
 
   .ant-empty-image {
     display: none;
   }
 `
 
-const DescriptionContainer = styled.div`
-  font-size: 12px;
-  margin-bottom: 24px;
-`
-
 type ActionsForComment = {
     updateAction?: (formValues: ITicketCommentFormState, obj: ITicketCommentUIState) => Promise<ITicketCommentUIState>,
     deleteAction?: (formValues: ITicketCommentFormState, obj: ITicketCommentUIState) => Promise<ITicketCommentUIState>,
 }
 
+const { TabPane } = Tabs
+
+const CommentsTabsContainer = styled.div<{ isTabsHidden: boolean }>`
+    padding: 0;
+    display: flex;
+    flex: 1 1 auto;
+
+    height: calc(100vh - 508px);
+    overflow-y: scroll;
+  
+    .ant-tabs-card.ant-tabs {
+      flex: 1 1 auto;
+
+      & > .ant-tabs-nav {
+        visibility: ${({ isTabsHidden }) => isTabsHidden ? 'hidden' : 'visible'};
```

Comment:
Why don’t use just ‘display: none’?

Reply:
`display: none` hides the element from DOM at all, but `visibility: hidden` keeps the element's size. Maybe this is the reason.

https://user-images.githubusercontent.com/25384290/165675517-0e949f36-8f8d-4f7f-ad55-00dbbc56d965.mov

Reply:
This is needed to display the animation.
Discussed with the design, now tabs are fixed when scrolling comments